doc/starting.rst

```diff
@@ -89,6 +89,28 @@ connection::
     pudb:6899: Please telnet into 127.0.0.1 6899.
     pudb:6899: Waiting for client...
 
+"Reverse" remote debugging
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In "reverse" remote debugging, pudb connects to a socket, rather than listening to one.
+
+First open the socket and listen using the netcat(``nc``), as below.
+Netcat of couse is not a telnet client, so it can behave diffrently than a telnet client.
+By using the ```stty``` with "no echo: and "no buffering" input options, we
+can make a socket that nonetheless behave simillarly::
+
+    stty -echo -icanon && nc -lcv 9999
```

Comment:
```suggestion
    stty -echo -icanon && nc -l -p 6899
```

Reply:
The version of netcat on my system wouldn't work without this change.

Reply:
It seems difference between mac and linux. 
https://superuser.com/questions/115553/netcat-on-mac-os-x

In mac `stty -echo  -icanon && nc -l 6899`
In linux `stty -echo -icanon && nc -l -p 6899` 

How about adding both of them?

Reply:
Yep, that sounds good.

Reply:
```suggestion
    stty -echo -icanon && nc -l -p 6898
    
When using the BSD version netcat that ships with MacOS, a server can be started like this::

    stty -echo -icanon && nc -l 6899
```